frontend/public/components/cluster-settings/related-objects.tsx

```diff
@@ -10,7 +10,12 @@ import {
 } from '../../module/k8s';
 import { ResourceLink, EmptyBox } from '../utils';
 
-const tableColumnClasses = ['', classNames('pf-m-hidden', 'pf-m-visible-on-sm'), ''];
+const tableColumnClasses = [
+  '', // Name
+  classNames('pf-m-hidden', 'pf-m-visible-on-sm'), // Resource
+  classNames('pf-m-hidden', 'pf-m-visible-on-sm'), //Group
```

Comment:
I think there are too many columns visible at `sm`. The wrapping is awkward. I'd suggest 

```suggestion
  classNames('pf-m-hidden', 'pf-m-visible-on-md'), // Group
```

We might want to put the group as muted text below the resource when the column is hidden.